src/RepoToolset/DefaultVersions.props

```diff
@@ -1,30 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
+
   <!--
     Sets versions of packages used by the toolset unless they were already specified by the repo.
   -->
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
   <PropertyGroup>
     <!-- Opt-out features -->
     <UsingToolXliff Condition="'$(UsingToolXliff)' == ''">true</UsingToolXliff>
     <UsingToolSourceLink Condition="'$(UsingToolSourceLink)' == ''">true</UsingToolSourceLink>
     <UsingToolXUnit Condition="'$(UsingToolXUnit)' == ''">true</UsingToolXUnit>
+    <UsingToolMicrosoftNetCompilers Condition="'$(UsingToolMicrosoftNetCompilers)' == ''">true</UsingToolMicrosoftNetCompilers>
```

Comment:
><UsingToolMicrosoftNetCompilers Condition="'$(UsingToolMicrosoftNetCompilers)' == ''">true</UsingToolMicrosoftNetCompilers> [](start = 4, length = 123)

I think we should keep this as a non-optional feature. ALl repos must use the Roslyn compilers. We should auto-detect when NetCore package is needed vs. Desktop package.

Reply:
All repos must use the Roslyn compilers. Not all repos must use the toolset compiler.

Reply:
Why must all repos use the package? If I'm getting a .NET Core SDK LKG, why can't I just use the compiler that comes with it. Why must I have two versions to maintain?

Reply:
Another thing: Microsoft.Net.Compilers.netcore nupkg edition won't be crossgen'ed and it will slow down core builds even more. I should be allowed to use the crossgen'ed copy that's bundled with .NET Core SDK.

Reply:
I see. Then do we ever need the NetCore package? Either repo can use desktop csc from Microsoft.Net.Compilers or it opts into using NetCore compilers. In the latter case we use the compilers bundled with .NET Core SDK.